### PR TITLE
Fix API auth and auto generate descriptions

### PR DESCRIPTION
**Pull request by GideonShils**
1. Updates auth type to use correct bearer token method. This now gets set automatically during "Try It" requests
2. Updates descriptions to be auto generated

- [x] Try it requests work
- [x] Auth is now properly documented
- [x] Descriptions show up properly still even after auto generation
- [x] Tested out new specs in spotlight elements to make sure things look good post generation: https://elements-demo.stoplight.io/?spec=https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

### CODE REVIEW

File: openapi-v2.yaml [hunk 1]
```diff
@@ -2,63 +2,35 @@ openapi: 3.0.3
 info:
   title: dbt Cloud API v2
   version: 2.0.0
-  description: |
```
Comment (MitchellBarker) [inline openapi-v2.yaml:5]: Lmk if I'm using the spotlight tool wrong. Looked like the PR description link was out of date, as it still had your 'gideon' namespace in the servers list. Think it's currently failing: https://elements-demo.stoplight.io/?spec=https://github.com/dbt-labs/dbt-cloud-openapi-spec/blob/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

**Review by MitchellBarker — COMMENTED** [review]
Ugh I thought I submitted this like an hour ago. My bad

File: openapi-v2.yaml (hunk 1 above)
Comment (GideonShils) [reply openapi-v2.yaml:5]: @MitchellBarker you have to paste in the "raw" file like this: https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

Comment (GideonShils) [reply openapi-v2.yaml:5]: Here's an updated link. Putting it in the description too: https://elements-demo.stoplight.io/?spec=https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/3d25048daf515be76a6a136965133e6f9c903084/openapi-v2.yaml

Comment (MitchellBarker) [reply openapi-v2.yaml:5]: Duh, my bad haha

**Review by MitchellBarker — APPROVED** [review]

**Pull request merged**